server/src/users/entities/user.entity.ts

```diff
@@ -19,7 +19,7 @@ export class User {
   @Column({ type: 'text', unique: true })
   email: string;
 
-  @Column('text', { select: false })
```

Comment:
adding this was what broke things
pw was being excluded in the user service select/get from db
therefore we couldn't check if pw was correct in auth service